src/Microsoft.VisualStudio.Editor.Razor/DefaultTagHelperResolver.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Razor.Extensions;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Razor;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal class DefaultTagHelperResolver : TagHelperResolver
+    {
+        private readonly ErrorReporter _errorReporter;
+
+        // Hack for testability. The view component visitor will normally just no op if we're not referencing
+        // an appropriate version of MVC.
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We don't need this attribute here. This is only visible in our own code anyway. This just makes working on the tests more annoying.